src/fu-tool.c

```diff
@@ -1133,6 +1133,74 @@ fu_util_update (FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_util_prompt_complete (priv->completion_flags, TRUE, error);
 }
 
+static gboolean
+fu_util_reinstall (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(FwupdRelease) rel = NULL;
+	g_autoptr(GPtrArray) rels = NULL;
+	g_autoptr(FuDevice) dev = NULL;
+
+	if (g_strv_length (values) != 1) {
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_INVALID_ARGS,
+				     "Invalid arguments");
+		return FALSE;
+	}
+
+	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_NONE, error))
+		return FALSE;
+
```

Comment:
It would be nice if we can support the device prompt `fwupdmgr` has here too, but not sure how hard that is to do.

Reply:
I can copy the code from `fwupdmgr`, I just though the maintenance burden would probably not be worth.

Reply:
Yeah I guess we don't have it in use for the other commands either right now, so big deal.